Mathlib/Data/QPF/Univariate/Basic.lean

```diff
@@ -53,7 +53,7 @@ class QPF (F : Type u → Type u) [Functor F] where
   abs : ∀ {α}, P α → F α
   repr : ∀ {α}, F α → P α
   abs_repr : ∀ {α} (x : F α), abs (repr x) = x
-  abs_map : ∀ {α β} (f : α → β) (p : P α), abs (f <$> p) = f <$> abs p
+  abs_map : ∀ {α β} (f : α → β) (p : P α), abs (P.map f p) = f <$> abs p
```

Comment:
Here you don't actually use the universe polymorphism, right?

Reply:
Yes. However, I think we should use simp-normal form as much as possible.